testing/BUILD.gn

```diff
@@ -77,4 +80,55 @@ if (current_toolchain == host_toolchain) {
       "//third_party/swiftshader_flutter:swiftshader",
     ]
   }
+
+  # All targets on all platforms should be able to the Metal utilities. On
+  # platforms where Metal in not available, the tests must be skipped or
```

Comment:
typo: "in not available" -> "is not available"

Reply:
Done.